src/client/api/extension.d.ts

```diff
@@ -257,8 +257,9 @@ export interface IExportedKernelService {
      * Gets a list of all kernel specifications that can be used to start a new kernel or to connect to an existing kernel.
      * Local, remote kernels are returned, including Python interpreters that
      * are treated as kernelspecs (as we can start Kernels for Python interpreters without Jupyter).
+     * @param {boolean} [refresh] Whether to force a refresh of the kernel specs (useful when dealing with remote kernels, as that list can change).
      */
-    getKernelSpecifications(): Promise<KernelConnectionMetadata[]>;
+    getKernelSpecifications(refresh?: boolean): Promise<KernelConnectionMetadata[]>;
```

Comment:
this allows us to refresh and view list of remote kernels.
Else today we only refresh the remote kernel when we start a new notebook.
Useful for external extensions to get latest remote kernels.